tox.ini

```diff
@@ -35,7 +35,7 @@ commands =
 description = Check code against coding style standards
 deps =
     black
-    flake8
+    flake8==4.0.1
```

Comment:
add `# flake8 version 5.0.0 & 5.0.1 has issues with `flake8.options.config``,

Reply:
Fixed in https://github.com/canonical/postgresql-operator/pull/15/commits/20aeab67061d337050c07ce9d13a939c6fc405db.